README.md

```diff
@@ -116,18 +116,59 @@ The tool to be used for formatting files can be configured with the following se
 "rubyLsp.formatter": "auto"
 ```
 
+#### Ruby version requirement
+
+By default, the Ruby LSP always uses the current project's Ruby version and bundle. This allows the LSP to find the
+correct gem versions automatically to index and to format the code matching behavior on CI.
```

Comment:
```suggestion
correct gem versions automatically so that index and formatting behaviour is consistent across all development and test environments.
```

Reply:
I don't think this says exactly what I meant. Indexing doesn't happen in CI, it's just formatting that needs to match it.

Reply:
Ok, how about:

> By default, the Ruby LSP uses the current project's Ruby version and bundle. This allows the LSP to index the correct gem versions, and to ensure formatting behaviour is consistent across all development and test environments.